api/golang/engine/lib/kurtosis_context/kurtosis_context.go

```diff
@@ -323,8 +371,24 @@ func runReceiveStreamLogsFromTheServerRoutine(
 }
 
 func newEnclaveContextFromEnclaveInfo(
```

Comment:
hopefully there aren't many CLI commands left that talk to the backend directly to do APIC stuff.

Reply:
Yup, I looked around quickly, couldn't find any. Once all those PRs are merged, I'll do an extensive testing of the different commands on a remote context, that will surely catch this.